### PR TITLE
[RFC] DOCS: Update options.txt for the removal of compatible

### PR DESCRIPTION
**Pull request by klusark**
This obviously shouldn't go in until #1622 and probably #228, but I figured I'd at least get it started. I also didn't want to just tack it onto one of those PRs as this one might need more work to get just right.

The text for the "compatible" option might need more work.This obviously shouldn't go in until #1622 and probably #228, but I figured I'd at least get it started. I also didn't want to just tack it onto one of those PRs as this one might need more work to get just right.

The text for the "compatible" option might need more work.

### CODE REVIEW

**Comment by Grimy** [comment]
This used to set Vi compatible mode, but has been removed and is now a no-op.

This is slightly misleading, since `set cp` is actually an error. I would rephrase it as such:

    Support for Vi-compatible mode has been removed from Neovim.
    "set nocp" is ignored; "set cp" is an error.

**Comment by Pyrohh** [comment]
@klusark do you mind if I finish this up for you?

**Comment by klusark** [comment]
@Pyrohh go ahead

**Comment by Pyrohh** [comment]
This can be closed.
